discord_slash/http.py

```diff
@@ -167,4 +185,3 @@ def delete(self, token, message_id="@original"):
         """
         req_url = f"/messages/{message_id}"
         return self.command_response(token, True, "DELETE", url_ending = req_url)
-
```

Comment:
Why was this line removed?

Reply:
I think my code editor auto-format the file such that there is one empty line at the end of the file. Looks like the file had 2 empty lines at the end that's why one line is removed. I can add back if you want?

Reply:
Oh oops sorry, for some reason i thought this removed the newline at the end of the file, which is required by pep8 ~~not that we follow it all the time anyways~~

Reply:
Ah I see, no worries all good!